src/solvers/dgsem_unstructured/dg_2d.jl

```diff
@@ -313,7 +313,7 @@ function calc_boundary_flux!(cache, t, boundary_condition::BoundaryConditionPeri
 end
 
 # Function barrier for type stability
-function calc_boundary_flux!(cache, t, boundary_conditions,
+function calc_boundary_flux!(cache, t, boundary_conditions::UnstructuredSortedBoundaryTypes,
```

Comment:
This seems to not be necessary - I added this for expressiveness

Reply:
I'm not sure whether we should add this here. It's not required and I'm not sure whether it helps here.